ui/panels-plugin/src/plugins/line-chart/tooltip/SeriesInfo.tsx

```diff
@@ -21,49 +21,112 @@ interface SeriesInfoProps {
   totalSeries: number;
 }
 
+// type ReplaceCharacters = '' | ':'
+
+// interface CharactersToReplace {
+//   ": ReplaceCharacters;
+//   =': ReplaceCharacters;
+// }
+
 function SeriesInfo(props: SeriesInfoProps) {
   const { seriesName, y, markerColor, totalSeries } = props;
-  // TODO (sjcobb): regex to remove __name__ and quotes, replace = with :
+  // TODO (sjcobb): regex to remove __name__ and quotes, replace = with :`
+  const formattedSeriesLabels = seriesName.replaceAll('"', '');
+
   if (totalSeries === 1) {
     return (
-      <Box sx={{ margin: '0 0 4px' }}>
+      <Stack spacing={0.5}>
         <Box
           sx={{
             height: '16px',
-            margin: '0 0 2px',
             display: 'flex',
             flexDirection: 'row',
             alignItems: 'center',
             justifyContent: 'left',
           }}
         >
           <SeriesMarker markerColor={markerColor} />
-          <Box component="span">value: {y}</Box>
+          <Box component="span" color="grey.300">
+            value:
+            <Box
+              component="span"
+              sx={{
+                color: 'white',
+                fontWeight: 700,
+                paddingLeft: '2px',
+              }}
+            >
+              {y}
+            </Box>
+          </Box>
         </Box>
-        <Box>{seriesName}</Box>
-      </Box>
+        <Divider sx={{ borderColor: 'grey.800', fontSize: 'body2' }} />
+        <Box>
+          {formattedSeriesLabels.split(',').map((name) => {
+            const [key, value] = name.split('=');
+
+            return (
+              <Box key={name} sx={{ display: 'flex', gap: '4px' }}>
+                <Typography fontSize="11px" color="grey.300">
+                  {key}:
+                </Typography>
+                <Typography
+                  sx={{
+                    color: 'white',
+                    fontSize: '11px',
+                    fontWeight: 700,
+                  }}
+                >
+                  {value}
+                </Typography>
+              </Box>
+            );
+          })}
+        </Box>
+      </Stack>
     );
   }
 
   return (
-    <Box sx={{ minHeight: '16px', margin: '0 0 2px', display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
-      <SeriesMarker markerColor={markerColor} />
+    <Box
+      sx={{
+        display: 'table-row',
+        paddingTop: 0.5,
+      }}
+    >
+      <Box
+        sx={{
+          display: 'table-cell',
+          maxWidth: '520px',
+        }}
+      >
+        <SeriesMarker markerColor={markerColor} />
+        <Box
+          component="span"
+          sx={{
+            color: 'grey.300',
+            display: 'inline-block',
+            maxWidth: '550px',
```

Comment:
Would it make sense to calculate this maxWidth relative to TOOLTIP_MAX_WIDTH (in case that constant changes in the future)?